courses/migrations/0012_readable_id_required.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.7 on 2019-06-11 19:21
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("courses", "0011_backpopulate_readable_id")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="course",
+            name="readable_id",
+            field=models.CharField(
+                max_length=255,
+                unique=True,
```

Comment:
There were some dupe readable id's in my database that caused the migration to fail (my fault), had to manually change them in Django admin then try again.  Just a heads up that might happen on CI/RC depending on what data was manually created for testing purposes.